### DISCUSSION TITLE
Support for negative grids and grid manipulation after instantiating a fieldset

### DISCUSSION THREAD

**Discussion by VeckoTheGecko** [Parcels-code/Parcels#1687] [Q&A] [answered]
## Question
@sruehs is working with model output from a MitGCM model where the depth is negative and the longitude is provided from 0 to 360 (instead of -180 to 180). 

Does Parcels natively support negative grids? (from what I've heard we only support positive grids?) What is the best way to go about manipulating the grid for the fieldset to make sure its what Parcels expects? Is the only way to use xarray as a pre-processing step? (which I don't think would be scalable to very large datasets?)

**Comment by erikvansebille** [comment, marked as answer]
No, Parcels certainly support negative grids. We do so in e.g. CROCO. As far as I know, there is no difference between positive and negative grids.
